fortran/example/README.md

```diff
@@ -0,0 +1,24 @@
+# Fortran example
+
+This is an example of how to use Spglib through the Fortran api
+
+## Examples of using spglib in a cmake project
+
+See the local [`CMakeLists.txt`](CMakeLists.txt) file for how to use
+spglib in a cmake project, using either native cmake options or pkg-config.
+
+## Examples of linking to spglib manually
+
+To manually compile `example.f90`:
+
+```console
+gfortran example.f90 -I[include directory] -L[library directory] -lsymspg
+```
+
+Here it is assumed that spglib library was built by
```

Comment:
Reference pkg-config command instead of cmake (if they have built it with cmake they would have necessary files to use cmake)

Reply:
I have no idea about pkg-config...

Reply:
Add something like:
> Where `[include directory]` and `[library directory]` are taken from `pkg-config --cflags --libs spglib_f08`

You can try manually:
```console
$ PKG_CONFIG_PATH=/path/to/spglib_f08.pc pkg-config --cflags --libs spglib_f08
```

Reply:
OK, thanks. It is done at c4a2e99.

Reply:
Thanks. It is done at db4b8c0.